orc8r/cloud/go/obsidian/swagger/poll_test.go

```diff
@@ -85,7 +85,7 @@ func Test_GetCombinedSwaggerSpecs(t *testing.T) {
 	defer registry.RemoveServicesWithLabel(orc8r.SwaggerSpecLabel)
 
 	for i, s := range services {
-		registerServicer(t, s, tags[i])
+		registerServicer(t, s, tags[i], swagger_lib.TagDefinition{})
```

Comment:
Side note: is this PR just on an old commit? Cause didn't we remove the whole swagger_lib thing?